I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 117, in script
    show crow right
Exception: Image 'crow' does not accept attributes 'right'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 117, in script
    show crow right
  File "/Users/kseniasaprykina/Downloads/renpy-8.1.3-sdk/renpy/ast.py", line 1428, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "/Users/kseniasaprykina/Downloads/renpy-8.1.3-sdk/renpy/ast.py", line 1391, in show_imspec
    renpy.config.show(name,
  File "/Users/kseniasaprykina/Downloads/renpy-8.1.3-sdk/renpy/exports.py", line 733, in show
    if not base.find_target() and renpy.config.missing_show:
  File "/Users/kseniasaprykina/Downloads/renpy-8.1.3-sdk/renpy/display/image.py", line 421, in find_target
    self.target = target._duplicate(a)
  File "/Users/kseniasaprykina/Downloads/renpy-8.1.3-sdk/renpy/display/core.py", line 499, in _duplicate
    args.extraneous()
  File "/Users/kseniasaprykina/Downloads/renpy-8.1.3-sdk/renpy/display/core.py", line 362, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'crow' does not accept attributes 'right'.

macOS-14.0-arm64-arm-64bit arm64
Ren'Py 8.1.3.23091805
knt day 1.0
Fri Oct 20 23:30:37 2023
